app/src/main/java/io/multy/ui/fragments/asset/TransactionInfoFragment.java

```diff
@@ -0,0 +1,280 @@
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.content.ContextCompat;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import io.multy.R;
+import io.multy.api.MultyApi;
+import io.multy.model.entities.TransactionHistory;
+import io.multy.model.entities.wallet.WalletAddress;
+import io.multy.model.responses.ServerConfigResponse;
+import io.multy.storage.RealmManager;
+import io.multy.ui.activities.BaseActivity;
+import io.multy.ui.fragments.BaseFragment;
+import io.multy.ui.fragments.WebFragment;
+import io.multy.util.Constants;
+import io.multy.util.CryptoFormatUtils;
+import io.multy.util.DateHelper;
+import io.multy.viewmodels.WalletViewModel;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+import static io.multy.util.Constants.TX_CONFIRMED_INCOMING;
+import static io.multy.util.Constants.TX_CONFIRMED_OUTCOMING;
+import static io.multy.util.Constants.TX_IN_BLOCK_INCOMING;
+import static io.multy.util.Constants.TX_IN_BLOCK_OUTCOMING;
+import static io.multy.util.Constants.TX_MEMPOOL_INCOMING;
+import static io.multy.util.Constants.TX_MEMPOOL_OUTCOMING;
+
+/**
+ * Created by anschutz1927@gmail.com on 16.01.18.
+ */
+
+public class TransactionInfoFragment extends BaseFragment {
+
+    public static final String TAG = TransactionInfoFragment.class.getSimpleName();
+    public static final String SELECTED_POSITION = "selectedposition";
+    public static final String TRANSACTION_INFO_MODE = "mode";
+    public static final String WALLET_INDEX = "walletindex";
+    public static final int MODE_RECEIVE = 1;
+    public static final int MODE_SEND = 2;
+
+    @BindView(R.id.linear_parent)
+    LinearLayout parent;
+    @BindView(R.id.toolbar_name)
+    TextView toolbarWalletName;
+    @BindView(R.id.back)
+    View buttonBack;
+    @BindView(R.id.text_date)
+    TextView textDate;
+    @BindView(R.id.image_operation)
+    ImageView imageOperation;
+    @BindView(R.id.text_value)
+    TextView textValue;
+    @BindView(R.id.text_coin)
+    TextView textCoin;
+    @BindView(R.id.text_amount)
+    TextView textAmount;
+    @BindView(R.id.text_money)
+    TextView textMoney;
+    @BindView(R.id.text_comment)
+    TextView textComment;
+    @BindView(R.id.text_addresses_from)
+    TextView textAdressesFrom;
+    @BindView(R.id.arrow)
+    ImageView imageArrow;
+    @BindView(R.id.logo)
+    ImageView imageCoinLogo;
+    @BindView(R.id.text_addresses_to)
+    TextView textAddressesTo;
+    @BindView(R.id.text_confirmations)
+    TextView textConfirmations;
+    @BindView(R.id.button_view)
+    TextView buttonView;
+    @BindView(R.id.donat_include)
+    View donatBlock;
```

Comment:
Where is your naming skills for last 3 views? viewDonateBlock etc. Also don't use russian transliterate words like "donat". Use donatE.